modules/openapi-generator/src/main/resources/kotlin-client/api.mustache

```diff
@@ -17,7 +17,7 @@ import {{packageName}}.infrastructure.Success
 import {{packageName}}.infrastructure.toMultiValue
 
 {{#operations}}
-class {{classname}}(basePath: kotlin.String = "{{{basePath}}}") : ApiClient(basePath) {
+{{>visibility}}class {{classname}}(basePath: kotlin.String = "{{{basePath}}}") : ApiClient(basePath) {
```

Comment:
All `{{>visibility}}` should be followed by a space. I see that there's a space in `visibility.mustache`, but if a user were to modify `visibility.mustache` to remove the tags, the trailing space may be removed (for example, if the user's git configuration removes trailing spaces). Rather than open to such bugs, we should aim for mustache files to generate valid code regardless of spacing in imported templates.

Reply:
The problem is that the space is only needed if the the class is internal, if it's `public` it's not needed.
And if I would add an extra space in the `public` case ktlint would fail with and error.

Maybe this it's easier to explain with an example of the generated code:
```
    internal class UserInternal() // note 4 spaces
     class UserPublic() // note 5 spaces ktlint error
```

Do you have any idea on how to fix this and get this output?
```
    internal class UserInternal() // note 4 spaces
    class UserPublic() // note 4 spaces
```

I'm not sure if I explained it right 😟

Reply:
If that's the case, I think it would be better to not have `visibility.mustache` and instead inline it like this:

```
{{#nonPublicApi}}internal {{/nonPublicApi}}class {{classname}}
```

This way there's not an additional layer of templating for people to change/remove `internal`, and it'll be more clear how the space applies at the usage site.

Reply:
Thanks for the suggestion, it's done 👍